app/models/test_track/web_session.rb

```diff
@@ -1,7 +1,4 @@
-require 'delayed_job'
-require 'delayed_job_active_record'
```

Comment:
I'm not sure why we had these here. We require `delayed_job` in the engine, so this seems unnecessary. I'll see if any apps blow up with this change before merging.